src/services/calendar/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const request = require('request-promise-native');
+const hooks = require('./hooks');
+
+const REQUEST_TIMEOUT = 4000; // in ms
+
+class Service {
+	constructor(options) {
+		this.options = options || {};
+	}
+
+	find(params) {
+		const userId = params.payload.userId;
+		const options = {
+			uri: 'https://schul-cloud.org:3000/events/',
+			headers: {
+				'Authorization': userId
```

Comment:
This line itself is fine, but we should change the calendar service to use better authorization than just the userId